scripts/evtx_dump_json.py

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env python3
+#   This file is part of python-evtx.
+#   Written by AJ Read with help from evtx_dump.py file written by Willi Ballenthin.
+#
+#   Purpose: User can dump evtx data into JSON format to either the command line or a JSON file in new line delimited format/JSON array.
+#   Details: The JSON object is created with only the EventRecordID from the System section of the evtx XML and all of the information within the EventData section.
+
+import Evtx.Evtx as evtx
+import Evtx.Views as e_views
+
+# Added packages
+import os
+import xmltodict
```

Comment:
please update `setup.py` with the new dependency on `xmltodict`

Reply:
Added to the install_requires section! 

` install_requires=[
            'six',
            'hexdump==3.3',
            'xmltodict==0.13.0', #added deps for evtx_dump_json.py script
               
            # pin deps for python 2, see #67
            'more_itertools==5.0.0',
            'zipp==1.0.0',
            'configparser==4.0.2',
            'pyparsing==2.4.7',
            ],`